app.py

```diff
@@ -147,6 +163,58 @@ def graph_refresh() -> None:
             return
 
 
+def graph_refresh_package_analyzer() -> None:
+    """Schedule refresh for packages that are not yet analyzed by package analyzer."""
+    graph = GraphDatabase()
+    graph.connect()
```

Comment:
Should we initialize `graph` instance here again and try to connect or maybe we should initialize it once somewhere like `__init__` because `graph_refresh` also initializes it?